recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python 3.8
```

Comment:
I don't think that's right: you're restricting this to *only* support Python 3.8 @mcarans ?

Reply:
@kynan I was following the instruction in the comment from conda-forge-linting which says "For the host section of the recipe, you should usually use python {{ python_min }} for the python entry.". Have I got it wrong?

Reply:
OK, hadn't seen this. Seems to be intended then, though I still find it odd...